.github/release.yml

```diff
@@ -1,8 +1,9 @@
 changelog:
   # Categorise according to what an instance manager needs to know
   categories:
-    # Posted in advance for #instance-managers
-    - title: "User-facing changes 👀"
+    # Add the right label if anything appears in here.
+    # Then re-generate the release notes.
+    - title: "❓❓❓ Uncategorised ❓❓❓"
```

Comment:
I like that the icons make it nice and clear to the release drafter to fix it up.

But I worry that one day someone will still miss it, and it will get published..
I'd probably just call it Uncategorised with zero or one icon, which would be more acceptable to publish. Am I over-thinking it?

Reply:
> Am I over-thinking it?

Maybe. It's possible to edit release descriptions after the initial release as well to fix up things. But I agree that the release managers should know the process well enough to not need those question marks and that it would be a bit funny to publish them. I don't mind either way.

@rioug, can you make a call on this?

Reply:
As Gaetan's offline until Monday, and neither of us feel strong, I will suggest we just stick with the current version and merge.